src/util/htmlToJson.ts

```diff
@@ -120,9 +120,9 @@ const handleBr = (nodes: HimalayaNode[], brIndex: number): HimalayaNode[] => {
 }
 
 /** Append children to parent as children property if it's necessary. */
-const joinChildren = (nodes: Block[]) => {
+const joinChildren = (nodes: Block[] | Block[][]) => {
   // split by chunk with size of 2, first element in chunk is Block - parent, the second is Block[] - children
-  const chunks = _.chunk(nodes, 2)
+  const chunks = _.chunk(nodes as Block[], 2)
```

Comment:
Why are we creating chunks of size 2 explicitly?

Reply:
I believe because `himalaya.parse` returns an array in which the first element is the parent and the second is the children.